javascript-bank/account.js

```diff
@@ -0,0 +1,50 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if (isNaN(amount)) {
+    return false;
+  } else if (!Number.isInteger(amount)) {
+    return false;
+  } else if (amount <= 0) {
+    return false;
+  } else if (typeof amount === 'string') {
+    return false;
```

Comment:
What if `amount` is a boolean or an array? It'd be better check that it _is_ a number (called "whitelisting") than check specific things that _aren't_ a number (called "blacklisting").